### ISSUE TITLE
Hi, wonder if there is a plan to support laravel 9.0 ?

### ISSUE THREAD

**Issue by symoo** [spiral/roadrunner-laravel#85]
> If you have any questions feel free to ask

laravel 9.0 released at 02/09. now I'm trying to upgrade got a problem.

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires laravel/framework ^9.0 -> satisfiable by laravel/framework[v9.0.0].
    - spiral/roadrunner-laravel v5.6.0 requires illuminate/view ~6.0 || ~7.0 || ~8.0 -> satisfiable by illuminate/view[v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.0].
    - Only one of these can be installed: illuminate/view[v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.0, v9.0.0], laravel/framework[v9.0.0]. laravel/framework replaces illuminate/view and thus cannot coexist with it.
    - Root composer.json requires spiral/roadrunner-laravel ^5.6 -> satisfiable by spiral/roadrunner-laravel[v5.6.0].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

seems like [roadrunner-laravel](https://github.com/spiral/roadrunner-laravel) is lock to 8.0 version.

**Comment by tarampampam** [comment]
Hi @symoo! Yea, PR #78 soon will be merged

**Comment by tarampampam** [comment]
:fire: Released - [v5.7.0](https://github.com/spiral/roadrunner-laravel/releases/tag/v5.7.0)
